draft-kb-capsule-conversion.md

```diff
@@ -98,14 +101,17 @@ If the intermediary receives any other valid response, it MUST NOT convert it to
 A Convertible Extended CONNECT request is a request that meets these criteria:
 
 * The method is "CONNECT".
-* A "capsule-protocol" header is present with an item value of "?1" (with or without parameters).
+* The request is known to use the Capsule Protocol, because:
```

Comment:
In addition to requiring the method to be CONNECT, should we also require the presence of a `:protocol` pseudo-header? I know the case of unextended CONNECT with `Capsule-Protocol = ?1` is invalid, but might as well be precise about the need for extended CONNECT.

Reply:
OK, done